k8s/production/custom/webhook-handler/deployments.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook-handler
+  namespace: custom
+  labels:
+    app: webhook-handler
+    svc: web
+spec:
+  selector:
+    matchLabels:
+      app: webhook-handler
+      svc: web
+      # TODO: evaluate
+  replicas: 2
```

Comment:
I tried reading the documentation, but I'm not sure if replicas does anything here? Would it be load balanced or something? Or should this be removed? @AlmightyYakob @mvandenburgh

Reply:
Yeah it would be load balanced. If there are multiple replicas, the assocated `Service` will dispatch requests to one randomly.

Reply:
Since this is set up behind a service, I think the effect would be that this load balances the incoming requests. According to [this](https://stackoverflow.com/questions/48789227/does-clusterip-service-distributes-requests-between-replica-pods) StackOverflow thread that's the behavior, although I couldn't find it explicitly mentioned in the docs as they claim. However, based on that thread and our past usage of this configuration, I believe this is the case.

All that being said, I think 2 is a fine value for this.